### DISCUSSION TITLE
Android View class android.support.v7.widget.AppCompatTextView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).

### DISCUSSION THREAD

**Discussion by richardhfj** [flutter-stripe/flutter_stripe#1883] [Q&A]
I am trying to implement the flutter_stripe plugin 11.0.0 into my app. It is currently functioning with my backend, functioning as expected on iOS. However, I am coming across several errors due to Android Theme.AppCompat issue. 
I have copied and pasted the styles.xml for values and values-night directly from the Stripe example on GitHub, however it seems I am running into some type of conflict. I am also using Flutter Native Splash, and copied and pasted for the values31 folders as well. 

styles.xml:
<?xml version="1.0" encoding="utf-8"?>
<resources>
    
    <style name="LaunchTheme" parent="Theme.AppCompat.Light.NoActionBar">
        
        <item name="android:windowBackground">@drawable/launch_background</item>
    </style>
    
    <style name="NormalTheme" parent="Theme.MaterialComponents">
        <item name="android:windowBackground">?android:colorBackground</item>
    </style>
</resources>

<?xml version="1.0" encoding="utf-8"?>
<resources>
    
    
    <style name="LaunchTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
        
        <item name="android:windowBackground">@drawable/launch_background</item>
    </style>
    
    <style name="NormalTheme" parent="Theme.MaterialComponents">
        <item name="android:windowBackground">?android:colorBackground</item>
    </style>
</resources>

However, this did not help.These are the errors that I am receiving. I am testing on a Samsung S21 Ultra as a physical device.Though my device did not crash from this, I am a bit concerned to leave this in production. Would be grateful for any advice. Thank you.

E/ThemeUtils(24710): View class android.support.v7.widget.AppCompatTextView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
E/ThemeUtils(24710): View class com.google.android.material.imageview.ShapeableImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
W/ResourcesCompat(24710): Failed to inflate ColorStateList, leaving it to the framework
W/ResourcesCompat(24710): java.lang.UnsupportedOperationException: Failed to resolve attribute at index 0: TypedValue{t=0x2/d=0x7c040084 a=-1}, theme={InheritanceMap=[id=0x7c1100a8com.google.android.gms:style/PayButtonDarkTheme], Themes=[com.google.android.gms:style/PayButtonDarkTheme, forced, android:style/Theme.DeviceDefault.Light.DarkActionBar, forced]}

**Comment by Shahidbangash** [comment]
yes , same here
